src/renderer/components/Login.js

```diff
@@ -117,12 +101,54 @@ class Login extends React.Component {
     ipcRenderer.send('login', { addr: login, mailPw: true })
   }
 
+  renderPasswordInput (keyShowPassword, keyValue) {
+    const tx = window.translate
+
+    let lockButton = (
```

Comment:
`const`

Reply:
you want a `let`? Help me :D

Reply:
I always want `const` :D (it's also commenting just beneath the row)

Reply:
But isn't const behaving differently on the scope? I use let always when its just temporary and only for this scope. But as soon as it's not changing you want a `const`?

Reply:
> But isn't const behaving differently on the scope?

No, `const` and `let` behave the same.

> But as soon as it's not changing you want a `const`?

Yep.